scripts/overlay_survey/simulation.py

```diff
@@ -231,11 +252,19 @@ def _getsurveyresult(self, params):
                                "count.",
                                node,
                                len(out_edges),
-                               node_json["numTotalOutboundPeers"])
-                node_json["numTotalOutboundPeers"] = len(out_edges)
+                               node_json["totalOutbound"])
+                node_json["totalOutbound"] = len(out_edges)
 
             _add_v2_survey_data(node_json)
 
+            # Rename peer count fields to match stellar-core's response
+            _rename_dictionary_field(node_json,
```

Comment:
Why do we need this rename? (previous version of code used numTotalInboundPeers/numTotalOutboundPeers so no rename was necessary from what I see)

Reply:
I changed the output fields in the graph to match the JSON output when I initially updated the script for the V2 survey. That means that `numTotalInboundPeers`/`numTotalOutboundPeers` becomes `totalInbound` and `totalOutbound`. I don't have a preference for which set of field names we use (that is, I could change it the other way around so JSON output becomes `numTotalInboundPeers`/`numTotalOutboundPeers`), but I do think the two outputs should have the same field names.